scaffold/Taskfile.yml

```diff
@@ -42,6 +42,11 @@ tasks:
     cmds:
       # @see https://architecture.lullabot.com/adr/20211026-one-time-login-first-drupal-user-account/
       - ./vendor/bin/drush user:unblock --uid=1
+      - |
+          [ $(drush core:requirements --severity=2 --format=tsv | wc -l) -lt 1 ] \
+          && echo "🟢 Status Report page checked" \
+          || false || echo "🔴 Status Report page check failed"
```

Comment:
What's this `|| false` for?

Reply:
You are right 🤔 
I guess this wasn't needed...
Let me try without it.